packages/react-ui-components/src/Tabs/tabs.tsx

```diff
@@ -171,6 +173,11 @@ export interface TabMenuItemProps {
      */
     tooltip?: string;
 
+    /**
+     * An optional notification count displayed as a badge.
+     */
+    notifications?: number;
```

Comment:
I'm curious if adding an extra prop for this is the only way to do it. Maybe we could allow `title` prop to take any elements, and then pass the badge in there. This way the Tabs component would stay more generic. Wdyt?

Reply:
@markusguenther what's your opinion about this?

Reply:
Yeah maybe it is more elegant to passthru the Badge to the title. If you need help with that just ping us @lorissikora  :)